sdk/core/azure_core_tracing_opentelemetry/examples/basic_usage.rs

```diff
@@ -0,0 +1,5 @@
+fn main() -> Result<(), Box<dyn std::error::Error>> {
```

Comment:
I'm assuming this is a placeholder, but should still probably have the copyright headers on all source files. (Another reason I should probably hook up prettier as opposed to / in addition to editorconfig).